sdk/messaging/azservicebus/client.go

```diff
@@ -37,6 +38,10 @@ type Client struct {
 		internal.NamespaceForAMQPLinks
 	}
 	retryOptions RetryOptions
+
+	// acceptNextTimeout controls how long the session accept can take before
+	// the server stops waiting.
+	acceptNextTimeout time.Duration
```

Comment:
Is this meant to be publicly configurable?  I didn't see where this is exposed.

Reply:
It's not publicly configurable for now. We have an open issue for it, but the functionality was useful for making some of these live tests practical (otherwise we have to wait for 1:05 for the timeout).